dats.json

```diff
@@ -154,6 +149,11 @@
 			"input/tosec/TOSEC-ISO/NEC PC-Engine CD*"
 		]
 	},
+	"database/metadat/tosec/NEC - PC-9801": {
```

Comment:
If we name them the same, then we won't have to change the database names elsewhere :wink: 

``` suggestion
	"database/metadat/tosec/NEC - PC-98": {
```

Reply:
I dont care, I thought we need to use the names according to TOSEC. If we do not, it is better.